test/testcmdlineparser.cpp

```diff
@@ -466,6 +467,23 @@ class TestCmdlineParser : public TestFixture {
         ASSERT_EQUALS("cppcheck: error: could not load cppcheck.cfg - not a valid JSON - syntax error at line 2 near: \n", logger->str());
     }
 
+    void checkVersion() {
```

Comment:
Please split this into separate functions. The test is not designed to have several invocations within a single test case.

Reply:
ok I have done it.